versioned_docs/version-3.0/releasenotes/release-3.0.0.md

```diff
@@ -4,3 +4,166 @@
+Production Insights
+
+**1. Operations-Friendly**
+
+- Cache TTL: In real-time scenarios, setting the Cache TTL ensures that data within the TTL provides low-latency queries from the local cache.
+
+- Seamless Scaling: One benefit of the compute-storage decoupled mode is the rapid elasticity of computing resources. Doris uses progressive active cache preheating technology, ensuring no impact on ongoing queries and imports during scalability periods.
+
+- Seamless Upgrades: During upgrades, the multi-process mode operation ensures ongoing queries and imports do not fail.
+
+
+
+**2. Load Isolation**
+
+- Multiple Computing Clusters: Different business characteristics (e.g., batch processing, real-time) or business units (different businesses within a group) can use independent computing clusters for physical isolation.
+
+- Read-Write Separation: Imports and queries can be completed by different computing clusters, supporting automatic (notification for preheating during import) and manual (periodic manual preheating) methods to preheat the query cluster's cache.
+
+
+
+**3. Positive Data Deletion**
+
+In distributed systems, common data deletion involves reverse GC, comparing files in shared storage with those recorded in metadata and deleting unrecorded files. This method can easily lead to data deletion errors under concurrent conditions. Relying on MetaService's transaction capabilities, Doris adopts asynchronous forward data deletion technology. Data generation is bound to a transaction, as is metadata deletion, and the asynchronous deletion process simply deletes data based on transaction records, eliminating the need for reverse GC.
+
+
+
+## Optimizer
+
+- All DQL and DML, except for load operations, will use the new optimizer. Default settings disable fallback to the old optimizer, which will be removed in subsequent versions.
+
+- Comprehensive top-N runtime filter applicable to all query modes, enhancing performance when a top-N operator is present.
+
+- Experimental Feature: Refactored logic for synchronized materialized view selection, transitioning from RBO to CBO to maintain consistency with asynchronous materialized views. If issues arise, the switch `set global enable_sync_mv_cost_based_rewrite = false` can be used to revert to RBO mode.
+
+- Experimental Feature: Partition-level statistics. By activating the switch `set global enable_partition_analyze = true`, partition-level statistics can be collected and utilized, improving query performance in cases of data partition skew.
+
+- Experimental Feature: Generated columns. When creating tables, columns can be designated as generated columns, which automatically compute results based on defined expressions at write time, offering more complex expressions compared to default values but disallowing explicit writing of specified values.
+
+## Execution Engine
+
+- Experimental Feature: Spill capability for core operators such as join, aggregate, and sort has reached GA usable status.
```

Comment:
It's unclear to the reader what this means... is it experimental? or GA? both?

I'd also link to the docs explaining.